### PR TITLE
Added icon

### PR DESCRIPTION
**Pull request by coliff**
Fixes: #39
If you'd like to suggest any changes, please let me know.
I think the path in `package.json` _might_ need to be different, but I'm not certain.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   },
   "publisher": "tombonnike",
   "author": "Tom Bonnike (bonniketom@gmail.com)",
+  "icon": "icon.png",
```
Comment (Tom-Bonnike) [inline package.json]: I think that‘s alright, that’s what Prettier does.

**Comment by Tom-Bonnike** [comment]
Hey, thank you for this PR! ❤️ 
Could you move the icon to the root of the project? That’s what `vscode-prettier` does (and then the path in `package.json` will be correct).
Can you also make sure the circle is perfect? It seems cut from the GitHub preview 

screenshot 2019-03-03 at 10 29 14

**Comment by coliff** [comment]
Hey. Thanks for the feedback. I’ll update the icon’s outer circle and the path in the package.json tomorrow morning.

**Comment by Tom-Bonnike** [comment]
Thaaanks! Just for clarity, the path in the package.json is correct but you put the icon in the `src` folder and it should be at the root instead.

**Comment by Tom-Bonnike** [comment]
Thank you!

**Pull request merged**